dex/testing/eth/harness.sh

```diff
@@ -114,13 +114,28 @@ cat > "${NODES_ROOT}/genesis.json" <<EOF
   "extradata": "0x00000000000000000000000000000000000000000000000000000000000000009ebba10a6136607688ca4f27fab70e23938cd0270000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
   "alloc": {
     "18d65fb8d60c1199bb1ad381be47aa692b482605": {
-        "balance": "26000000000000000000000"
+        "balance": "260000000000000000000000"
     },
     "4f8ef3892b65ed7fc356ff473a2ef2ae5ec27a06": {
-        "balance": "11000000000000000000000"
+        "balance": "110000000000000000000000"
     },
     "946dfaB1AD7caCFeF77dE70ea68819a30acD4577": {
-        "balance": "11000000000000000000000"
+        "balance": "110000000000000000000000"
+    },
+    "dd93b447f7eBCA361805eBe056259853F3912E04": {
+        "balance": "110000000000000000000000"
+    },
+    "1D4F2ee206474B136Af4868B887C7b166693c194": {
+        "balance": "110000000000000000000000"
+    },
+    "946dfaB1AD7caCFeF77dE70ea68819a30acD4577": {
+        "balance": "110000000000000000000000"
+    },
+    "3e983c7e08d22CaB842Bd54111087E09C578b3c3": {
+        "balance": "110000000000000000000000"
+    },
+    "653445e4C2f48B370fE10e59940E5e9Cb853D03F": {
+        "balance": "110000000000000000000000"
```

Comment:
What's the deal with these again? They derive from a seed used in testing somewhere? Why not just give it all to alpha and request as needed?

Reply:
At least two are from the simnet testing program and I believe the other are the harness nodes. Sure can give all to alpha and send if that's less confusing. I should have commented better here either way.